src/app/views/common/popups/PanelWrapper.tsx

```diff
@@ -50,60 +60,48 @@ export function PanelWrapper(props: WrapperProps) {
 
   const onRenderHeader = (): JSX.Element => (
     <div style={{ display: 'flex', alignItems: 'center' }}>
-      <IconButton
-        iconProps={{ iconName: 'Back' }}
-        ariaLabel={translateMessage('Back')}
+      <Button
+        icon={< ArrowStepBack20Regular />}
+        aria-label={translateMessage('Back')}
         onClick={() => dismissPopup()}
-        styles={{ root: { marginRight: 8 } }}
       />
-      <span style={{ fontSize: theme.fonts.xLarge.fontSize, fontWeight: theme.fonts.xLarge.fontWeight }}>
+      <span>
         {title}
       </span>
     </div>
   );
 
   return (
     <div>
-      <Panel
-        isOpen={isOpen}
-        hasCloseButton={false}
-        type={panelType}
-        headerText={headerText.toString()}
-        isFooterAtBottom={true}
-        isBlocking={true}
-        isLightDismiss={false}
-        closeButtonAriaLabel='Close'
-        overlayProps={panelOverlayProps}
-        onRenderFooterContent={onRenderFooterContent}
-        onRenderHeader={showBackButton ? onRenderHeader: undefined}
+      <Drawer
+        open={isOpen}
+        type='overlay'
+        size='full'
       >
-        <IconButton
-          styles={{
-            root: {
-              float: 'right',
-              zIndex: 1,
-              marginTop: -30
-            }
-          }}
-          iconProps={{ iconName: 'Cancel' }}
-          ariaLabel={translateMessage('Close')}
-          onClick={() => dismissPopup()}
-        />
-        <div>
-          {
-            <Suspense fallback={<Spinner />}>
-              <Component
-                {...popupsProps}
-                data={popupsProps.data || {}}
-
-                dismissPopup={() => dismissPopup()}
-                closePopup={(e: any) => closePopup(e)}
-              />
-            </Suspense>
-          }
-        </div>
-
-      </Panel>
+        <DrawerHeader>
+          {showBackButton && onRenderHeader()}
+          <Button
```

Comment:
I suggest using the `<DrawerHeaderTitle>` component to display titles and it also has the `aside{}` prop for the dismiss function
You can also use the `Dismiss24Regular` icon